src/main/java/seedu/address/logic/commands/itemcommand/ItemListCommand.java

```diff
@@ -19,10 +18,9 @@
 
 
     @Override
-    public CommandResult execute(Model model) throws CommandException {
-        requireNonNull(model);
-        requireInventoryModel(model);
-        InventoryModel inventoryModel = (InventoryModel) model;
+    public CommandResult execute(InventoryModel inventoryModel, DeliveryModel deliveryModel) {
+        requireNonNull(inventoryModel);
```

Comment:
Should it check for deliveryModel to be nonNull too?

Reply:
I know we're changing this, but to answer this, I didn't think it necessary to check deliveryModel since it wasn't being used